### PR TITLE
Add pull reminder badge to README

### LINKED ISSUE #491: docs: add pull reminders badge to README
This is allows us to use the pull reminders slack integration for free.

### PR DESCRIPTION
**Pull request by ableuler**
Closes #491.

### CODE REVIEW

**Comment by ableuler** [comment]
@jirikuncar Please go ahead and merge or add me to the respective maintainers group.

File: README.rst [hunk 1]
```diff
@@ -38,6 +38,10 @@
 .. image:: https://img.shields.io/github/license/SwissDataScienceCenter/renku-python.svg
    :target: https://github.com/SwissDataScienceCenter/renku-python/blob/master/LICENSE
 
+.. image:: https://pullreminders.com/badge.svg
+   :target: https://pullreminders.com?ref=badge
+   :alt: Pull reminders
```
Comment (jirikuncar) [inline README.rst]:
```suggestion
   :alt: Pull reminders
   :align: right
```

Comment (jirikuncar) [reply README.rst]: @ableuler do we want to keep it right as in the other repos?

Comment (ableuler) [reply README.rst]: I tried, but rst insisted on putting it onto a new line, so I thought left-justified looks better.

**Pull request merged**